### ISSUE TITLE
JOSS: Diffuse Interface Method

### ISSUE THREAD

**Issue by Alex-Vasile** [uw-comphys/opencmp#7]
1. arxiv "article shares a lot of the authors but the software OpenCMP is not mentioned. Did you use a different implementation there? If not, I think it would be a good idea to cross-reference your papers. In the present joss paper a clear reference to a more physics and simulation results focused paper would help the interested reader to find details of your work."
2. "using the diffuse interface method to represent immersed boundaries is not a new idea presented for the first time in arxiv:2101.06824, right? Perhaps you should add a reference to the paper which indicates this and use your reference to refer the reader to the improvements you made to the method."
3. "You mention that there are no other implementations of the diffuse interface method. Is that really correct? How about the Phase Field implementation of comsol? Can that not be used to simulate "hard" boundaries by choosing super high values for density and viscosity or so?"

per: https://github.com/openjournals/joss-reviews/issues/3742#issuecomment-940427034

**Comment by WilkAndy** [comment]
Monte(2021) is published: add the DOI for https://onlinelibrary.wiley.com/doi/epdf/10.1002/cjce.24320 to the JOSS paper

**Comment by nasserma** [comment]
Responding to these points in order:

1. We used a different implementation in Monte(2021), which preceded the one in OpenCMP, which we have clarified in the most recent revision of the paper, along with updating the reference indicated publication.
2.  We added an additional reference for the diffuse interface method (with stabilization) that we used Nguyen(2018). Note that a review article on immersed boundary methods is also included Mittal(2005).
3. To our knowledge, this is the only open-source implementation of the diffuse interface method with stabilization for Dirichlet boundary conditions (see Nguyen(2018)). What you are describing with Comsol's phase field model would not straightforward to implement, would not be stable when using Dirichlet BCs, and is not open-source.
